packages/react/src/components/RadioTile/RadioTile.tsx

```diff
@@ -81,59 +93,19 @@ const RadioTile = React.forwardRef(function RadioTile(
 });
 
 RadioTile.propTypes = {
-  /**
-   * `true` if this tile should be selected.
-   */
   checked: PropTypes.bool,
-
-  /**
-   * The tile content.
-   */
   children: PropTypes.node,
-
-  /**
-   * The CSS class names.
-   */
   className: PropTypes.string,
-
-  /**
-   * Specify whether the RadioTile should be disabled
-   */
   disabled: PropTypes.bool,
-
-  /**
-   * The ID of the `<input>`.
-   */
   id: PropTypes.string,
-
-  /**
-   * `true` to use the light version. For use on $ui-01 backgrounds only.
-   * Don't use this to make tile background color same as container background color.
-   */
   light: deprecate(
     PropTypes.bool,
     'The `light` prop for `RadioTile` is no longer needed and has ' +
       'been deprecated in v11 in favor of the new `Layer` component. It will be moved in the next major release.'
   ),
-
-  /**
-   * The `name` of the `<input>`.
-   */
   name: PropTypes.string,
-
-  /**
-   * The handler of the massaged `change` event on the `<input>`.
-   */
   onChange: PropTypes.func,
-
-  /**
-   * Specify the tab index of the wrapper element
-   */
   tabIndex: PropTypes.number,
-
-  /**
-   * The `value` of the `<input>`.
-   */
```

Comment:
All these comments must remain, they're used by storybook to annotate the prop tables in the storybook docs pages.

Reply:
Thanks for the feedback , done the requested changes